omni-relayer/src/utils/near.rs

```diff
@@ -46,39 +46,35 @@ pub async fn handle_streamer_message(
             Nep141LockerEvent::InitTransferEvent {
                 ref transfer_message,
             }
-            // TODO: Later it's better to add a separate key in db for storing events with any
-            // troubles there. For now we just update whole event with a new fee for the same nonce
             | Nep141LockerEvent::UpdateFeeEvent {
                 ref transfer_message,
             } => {
-                // TODO: If fee is insufficient, it should be handled later. For example,
-                // add to redis and try again in 1 hour
                 match utils::fee::is_fee_sufficient(
                     jsonrpc_client,
                     &transfer_message.sender,
                     &transfer_message.recipient,
                     &transfer_message.token,
-                    transfer_message.fee.into(),
+                    transfer_message.fee.fee.into(),
```

Comment:
Actually, I think it would be a good idea to pass the native fee here as well.

Reply:
I want to make all changes related to fee calculation in the next PR. This whole function must be rewritten from scratch as well as its signature